lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -60,14 +60,14 @@ typedef dynamic Dispatcher(dynamic action);
 ///
 /// __Example:__
 /// ```dart
-///     UiFactory<CounterProps> ConnectedCounter = connect<CounterState, CounterProps>(
-///         mapStateToProps: (state) => (
-///           Counter()..count = state.count
-///         ),
-///         mapDispatchToProps: (dispatch) => (
-///           Counter()..increment = () => dispatch(INCREMENT_ACTION())
-///         ),
-///     )(Counter);
+///     UiFactory<CounterProps> Counter = connect<CounterState, CounterProps>(
+///       mapStateToProps: (state) => (Counter()
+///         ..count = state.count
+///       ),
+///       mapDispatchToProps: (dispatch) => (Counter()
+///         ..increment = (() => dispatch(INCREMENT_ACTION()))
+///       ),
+///     )(_$Counter);
```

Comment:
#supernit while we're here we could improve the formatting a little more 🤷‍♂ 
```suggestion
/// UiFactory<CounterProps> Counter = connect<CounterState, CounterProps>(
///   mapStateToProps: (state) => (Counter()
///     ..count = state.count
///   ),
///   mapDispatchToProps: (dispatch) => (Counter()
///     ..increment = (() => dispatch(INCREMENT_ACTION()))
///   ),
/// )(_$Counter);
```